contents/blog/openlineage-egeria/index.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: OpenLineage Support in Egeria
+date: 2022-04-15
+author: Mandy Chessel
+image: ./image.svg
+banner: ./banner.svg
+description: The Egeria project uses OpenLineage to enhance its production of holistic metadata about an organization's operations.
+---
+## OpenLineage Support in Egeria 
+
+[Egeria](https://egeria-project.org) is a sister open source project to OpenLineage in the LF AI and Data Foundation. Egeria provides Open Metadata and Governance standard types and integration technology to exchange metadata between different technologies. It stitches together different standards to create a complete landscape of metadata about an organization’s digital operations.
+OpenLineage is very welcome to the Egeria team since it defines a standard for dynamic lineage capture.  This means Egeria can capture open lineage events to detect new assets and activity around them, link this new knowledge into the existing metadata and distribute it to the open metadata ecosystem.
```

Comment:
The Gatsby markdown parser needs a blank line in between paragraphs here (and the following three) or they won't be separated by any space in the final page.

Reply:
Thanks -- made the change.